### PR TITLE
Fix utf8

### PR DESCRIPTION
**Pull request by douglaz**

### CODE REVIEW

File: src/main/scala/ignition/core/jobs/utils/SparkContextUtils.scala [hunk 1]
```diff
@@ -153,9 +154,9 @@ object SparkContextUtils {
         processTextFiles(paths, minimumPaths)
     }
 
-    private def stringHadoopFile(paths: Seq[String], minimumPaths: Int): RDD[String] = {
+    private def stringHadoopFile(paths: Seq[String], minimumPaths: Int): RDD[Try[String]] = {
       processPaths((p) => sc.sequenceFile(p, classOf[LongWritable], classOf[org.apache.hadoop.io.BytesWritable])
-                .map({ case (k, v) => new String(v.getBytes, 0, v.getLength) }), paths, minimumPaths)
+                .map({ case (k, v) => Try { ByteUtils.toString(v.getBytes, 0, v.getLength, "UTF-8") } }), paths, minimumPaths)
```
Comment (fparisotto) [inline src/main/scala/ignition/core/jobs/utils/SparkContextUtils.scala]: #Minor, why not ByteUtils.toString(v.getBytes, "UTF-8") ?

**Comment by fparisotto** [comment]
LGTM!

File: src/main/scala/ignition/core/jobs/utils/SparkContextUtils.scala (hunk 1 above)
Comment (luisguilherme) [inline src/main/scala/ignition/core/jobs/utils/SparkContextUtils.scala]: :+1:

**Comment by luisguilherme** [comment]
LGTM

**Pull request merged**